shim/interfaces.go

```diff
@@ -346,6 +346,10 @@ type ChaincodeStubInterface interface {
 	// available within the transaction in the committed block regardless of the
 	// validity of the transaction.
 	SetEvent(name string, payload []byte) error
+
+	// GetMSPID returns the local mspid of the peer by checking the CORE_PEER_LOCALMSPID
+	// env var and returns an error if the env var is not set.
+	GetMSPID() (string, error)
```

Comment:
This is exactly and explicitly what we do not want. This incompatibly changes the shim contract.